gallery/src/components/NxForm/NxFormExample.tsx

```diff
@@ -117,6 +119,9 @@ export default function NxFormExample() {
                     submitError={submitError}
                     validationErrors={validationErrors}
                     submitMaskState={submitMaskState}>
+      <NxTile.HeaderSubtitle>
```

Comment:
This is somewhat problematic, but it's tough to say what the right way forward would be since this example is of a form "in isolation" without any particular wrapping context.  Using `HeaderSubtitle` to wrap the notice would be correct if the form was in a tile (such that it is the entire tile).  That's not the case here.  You might want to ask Design what this message should look like on a form that's "in abstract" as opposed to being in any particular type of container.